markets/perps-market/contracts/modules/AsyncOrderModule.sol

```diff
@@ -121,6 +121,26 @@ contract AsyncOrderModule is IAsyncOrderModule {
         _settleOrder(offchainPrice, order, settlementStrategy);
     }
 
+    function getOrder(
+        uint128 marketId,
+        uint128 accountId
+    ) public view override returns (AsyncOrder.Data memory) {
+        return PerpsMarket.loadValid(marketId).asyncOrders[accountId];
+    }
+
+    function cancelOrder(uint128 marketId, uint128 accountId) external override {
+        AsyncOrder.Data storage order = PerpsMarket.loadValid(marketId).asyncOrders[accountId];
+        if (order.sizeDelta == 0) {
+            revert OrderDoesNotExists(marketId, accountId);
+        }
```

Comment:
there is a function on order called `checkValidity` which does this check for ya